src/jsonv-tests/serialization_builder_tests.cpp

```diff
@@ -472,13 +472,15 @@ class wrapped_things
 
 TEST(serialization_builder_access_mutate)
 {
+#ifndef _MSC_VER
```

Comment:
Interesting that MSVC can't compile this.

Reply:
I am not sure. I didn't try to dig deeper. Since everything else was built fine and tests run successfully I just excluded it from MSVC build.